public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/sessions.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { format } from '@wordpress/date';
+import { Fragment } from '@wordpress/element';
+
+//todo all sessions and also time slot headers, so maybe rename?
+export function Sessions( { sessions } ) {
+	const sessionsGroupedByTime = groupSessionsByTime( sessions );
+	const timeGroups            = [];
+
+	for ( const timestamp of Object.keys( sessionsGroupedByTime ) ) {
+		const gridRow = `
+			time-${ format( 'Hi', sessionsGroupedByTime[ timestamp ][0].derived.startTime ) } /
+			time-${ format( 'Hi', sessionsGroupedByTime[ timestamp ][0].derived.endTime ) }
+		`;
+
+		timeGroups.push(
+			<h3
```

Comment:
Reference: https://css-tricks.com/building-a-conference-schedule-with-css-grid/#article-header-id-1

I think they are very much necessary.

Reply:
Ah, I see. I was thinking each session would have a rowheader/columnheader for this kind of information (which is also why I thought the tracks shouldn't be hidden). I think this works, but we might want to check that it's not overly verbose for screen reader users.

Reply:
For example, having each individual session in a heading makes navigating by headings very time-consuming, since you need to move through each track to get past the schedule.